gulp-tasks/copy.js

```diff
@@ -0,0 +1,22 @@
+const gulp = require('gulp');
+
+const extensions = [
+  'json',
+];
+
+const extensionsGlob = () => {
+  if (extensions.length === 1) {
+    return `${global.__buildConfig.src}/**/*.${extensions[0]}`
+  } else {
+    return `${global.__buildConfig.src}/**/*.{${extensions.join(',')}}`
```

Comment:
`.{json}` doesn't work? If it does, if/else isn't needed.

Reply:
It doesn't work :(